addons/account_edi_ubl_cii/models/account_edi_xml_ubl_20.py

```diff
@@ -314,7 +314,7 @@ def _get_invoice_line_vals(self, line, taxes_vals):
         total_fixed_tax_amount = sum([
             vals['amount']
             for vals in allowance_charge_vals_list
-            if vals['allowance_charge_reason_code'] == 'AEO'
+            if vals.get('charge_indicator')
```

Comment:
the `== 'AEO'` is there for a reason :D please keep it

Reply:
Hey @JulienVR, this choice was made based on a discussion with @smetl :) (to keep the report consistent and keep the business logic consistent) 

Do you see a reason why it would not work as expected ?

Reply:
Nevermind, the fix was right ! Sorry :)